Makefile

```diff
@@ -14,7 +14,11 @@ HOOK_BIN_DIR ?= ${PREFIX}/libexec/oci/hooks.d
 ETCDIR ?= /etc
 HOOK_DIR ?= ${PREFIX}/share/containers/oci/hooks.d/
 
-all:
+.PHONY: docs
+docs:
+	$(MAKE) -C docs
+
+all: docs
 	$(GO_BUILD) -o bin/oci-seccomp-bpf-hook $(PROJECT)
```

Comment:
Can we add a new `build` target to avoid recompiling the docs when we want to build?

Reply:
Fixed.  Added binary